x11-wm/cwm/cwm-5.6.ebuild

```diff
@@ -1,42 +0,0 @@
-# Copyright 1999-2018 Gentoo Authors
```

Comment:
You're removing last stable version?

Reply:
Yes, I don't see the point in supporting an ebuild with memory leaks and upstream has deemed dead for > 4 years

Reply:
I suggest stabilizing the new version first. Either 6.6 or 6.7

https://devmanual.gentoo.org/ebuild-maintenance/removal/index.html

Reply:
6.6 stabilize sounds good.